src/backend/distributed/commands/collation.c

```diff
@@ -278,7 +278,7 @@ PreprocessDropCollationStmt(Node *node, const char *queryString)
 								(void *) dropStmtSql,
 								ENABLE_DDL_PROPAGATION);
 
-	return NodeDDLTaskList(ALL_WORKERS, commands);
+	return NodeDDLTaskList(ALL_NON_COORD_WORKERS, commands);
```

Comment:
will rebase onto https://github.com/citusdata/citus/pull/4003 and rename this as ALL_NON_COORDINATOR_WORKERS